ElementaryTasks/Arrays.cs

```diff
@@ -49,5 +49,109 @@ public void SumOfElements()
             }
             Console.WriteLine(sum);
         }
-    }   
+
+        public void MostCommonValue()
```

Comment:
Methods names must be verbs (e.g. GetSomething, SetSomething etc)

Reply:
renamed to "FindMostCommonValue"